packages/core/src/network.js

```diff
@@ -31,7 +31,7 @@ export class Network {
 
     if (this.interceptEnabled) {
       this.onRequest = createRequestHandler(this, options.intercept);
-      this.onRequestFinished = createRequestFinishedHandler(this, options.intercept);
+      this.onRequestFinished = createRequestFinishedHandler(this, options.authorization, options.intercept);
```

Comment:
No need to pass in extra arguments here. The instance is already passed as the first argument and also already has an `.authorization` property (see 10 lines above).